code/modules/organs/internal/heart.dm

```diff
@@ -170,7 +172,9 @@
 			if(PULSE_2FAST, PULSE_THREADY)
 				blood_max *= 1.5
 
-		if(CE_STABLE in owner.chem_effects)
+		if(CE_BLOODCLOT in owner.chem_effects)
+			blood_max *= 0.5
```

Comment:
Too much for a chem so widely available and easy to make. I would say 0.8x at the maximum.

Reply:
Heating reagents requires a bit more preparation; maybe cold tricordrazine (which is less effective for healing) and hot cough syrup produce a more effective form?

Reply:
CE_STABLE already does 0.8, and that's even easier to acquire. i raised coagzolug to 0.7 to get this PR merged